content/realtime/connection.textile

```diff
@@ -94,7 +94,7 @@ realtime.connection.on(.connected) { change in
   );
 ```
 
-<span lang="ruby">Note that all examples on this page assume you are running them within an EventMachine reactor. Find out more in our "Realtime usage documentation":/realtime/usage.</span>
+<span lang="ruby">Note that all examples on this page assume you are running them within an EventMachine. Find out more in our "Realtime usage documentation":/realtime/usage.</span>
```

Comment:
I removed reactor here as I thought reactor for EventMachine just meant the type of pattern it uses and not a specific aspect of EventMachine and that because we've used Reactor historically best to remove the term for clarity.

However, I'm not an expert on EventMachine so happy to add it back in if you think it makes sense.

Reply:
It is, but if we're going to consider removing it then I think we should discuss that in a separate issue personally and leave this PR to removing "reactor" as a product. WDYT?